templates/packer/aws/author-publish-dispatcher.json

```diff
@@ -109,9 +112,9 @@
       "inline": [
         "set -o nounset",
         "set -o errexit",
-        "export PATH=/opt/puppetlabs/puppet/bin/:/sbin:/usr/sbin:/usr/local/sbin:${PATH}",
+        "export PATH={{user `inspec_dir`}}:/sbin:/usr/sbin:/usr/local/sbin:${PATH}",
```

Comment:
This `PATH` doesn't seem to be necessary anymore since the `inspec` bin has a full path and then root will have its own `PATH` anyway. Please verify, if not needed anymore, please remove this PATH exporting.

Reply:
Addition of inspec bin path to `PATH` variable is not required, if we don't call/use inspec anywhere else and it doesn't seems to be used anywhere else other than this. So it can be removed.